src/bun.js/module_loader.zig

```diff
@@ -2335,165 +2335,208 @@ pub const HardcodedModule = enum {
-            .{ "tty", .{ .path = "node:tty" } },
-            .{ "url", .{ .path = "node:url" } },
-            .{ "util", .{ .path = "node:util" } },
-            .{ "util/types", .{ .path = "node:util/types" } },
-            .{ "v8", .{ .path = "node:v8" } },
-            .{ "vm", .{ .path = "node:vm" } },
-            .{ "wasi", .{ .path = "node:wasi" } },
-            .{ "worker_threads", .{ .path = "node:worker_threads" } },
-            .{ "zlib", .{ .path = "node:zlib" } },
-
-            // It implements the same interface
-            .{ "sys", .{ .path = "node:util" } },
-            .{ "node:sys", .{ .path = "node:util" } },
-            .{ "inspector/promises", .{ .path = "node:inspector" } },
-            .{ "node:inspector/promises", .{ .path = "node:inspector" } },
-
-            // These are returned in builtinModules, but probably not many packages use them
-            // so we will just alias them.
-            .{ "_http_agent", .{ .path = "node:http" } },
-            .{ "_http_client", .{ .path = "node:http" } },
-            .{ "_http_common", .{ .path = "node:http" } },
-            .{ "_http_incoming", .{ .path = "node:http" } },
-            .{ "_http_outgoing", .{ .path = "node:http" } },
-            .{ "_http_server", .{ .path = "node:http" } },
-            .{ "_stream_duplex", .{ .path = "node:stream" } },
-            .{ "_stream_passthrough", .{ .path = "node:stream" } },
-            .{ "_stream_readable", .{ .path = "node:stream" } },
-            .{ "_stream_transform", .{ .path = "node:stream" } },
-            .{ "_stream_writable", .{ .path = "node:stream" } },
-            .{ "_stream_wrap", .{ .path = "node:stream" } },
-            .{ "_tls_wrap", .{ .path = "node:tls" } },
-            .{ "_tls_common", .{ .path = "node:tls" } },
-
-            // Older versions of `readable-stream` is incompatible with latest
-            // version of Node.js Stream API, which `bun` implements
-            // .{ "readable-stream", .{ .path = "node:stream" } },
-            // .{ "readable-stream/consumer", .{ .path = "node:stream/consumers" } },
-            // .{ "readable-stream/web", .{ .path = "node:stream/web" } },
-
-            // Thirdparty packages we override
-            .{ "@vercel/fetch", .{ .path = "@vercel/fetch" } },
-            .{ "detect-libc", .{ .path = "detect-libc" } },
-            .{ "detect-libc/lib/detect-libc.js", .{ .path = "detect-libc" } },
-            .{ "isomorphic-fetch", .{ .path = "isomorphic-fetch" } },
-            .{ "node-fetch", .{ .path = "node-fetch" } },
-            .{ "undici", .{ .path = "undici" } },
-            .{ "utf-8-validate", .{ .path = "utf-8-validate" } },
-            .{ "ws", .{ .path = "ws" } },
-            .{ "ws/lib/websocket", .{ .path = "ws" } },
-        },
-    );
+
+    pub const Aliases = struct {
+        // Used by both Bun and Node.
+        const CommonAliases = bun.ComptimeStringMap(
```

Comment:
can we make this two separate ComptimeStringMap? That way there's only one hash table lookup for each target, instead of two. The `++` operator should work to combine two comptime-known arrays of tuples

Reply:
yes, fixed.